portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,37 +33,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
This is a good time to update the javadoc to remove the TODO and describe what the class does not.

That is one downside to comments, that they are too easy to get out of sync with the actual code and when that happens they can be more confusing than helpful.